docs/topics/db/models.txt

```diff
@@ -1232,6 +1232,11 @@ provide any connection between the rows in the different database tables. A
 proxy model can inherit from any number of abstract model classes, providing
 they do *not* define any model fields.
 
+.. versionchanged:: 1.10
```

Comment:
Put this annotation below the paragraph and describe the behavior in older versions below it.